app/models/katello/content_view_version_export_history.rb

```diff
@@ -2,6 +2,8 @@ module Katello
   class ContentViewVersionExportHistory < Katello::Model
     include Authorization::ContentViewVersionExportHistory
     audited except: :metadata
+    delegate :organization, to: :content_view_version
+    delegate :id, to: :organization, prefix: true
```

Comment:
This is the same as 
```rb
def organization_id
  organization.id
end
```
Rubocop didn't like when I tried to do the above, so I discovered that I can use `delegate` with `prefix: true` and it has the same effect.

Reply:
wow